chacra/controllers/repos/projects.py

```diff
@@ -0,0 +1,55 @@
+from pecan import expose, abort, request
+from chacra.models import Project
+from chacra.models.repos import Repo
+from chacra import models
+from chacra.controllers import error
+from chacra.controllers.repos.refs import RefController
+
+
+class ProjectController(object):
+
+    def __init__(self, project_name):
+        self.project_name = project_name
+        self.project = Project.query.filter_by(
+            name=project_name
+        ).join(Repo).filter(
+            Repo.path != None
+        ).first()
+        if not self.project:
+            if request.method != 'POST':
+                abort(404)
+            elif request.method == 'POST':
```

Comment:
Agreed, I don't think we should either. This was cruft from porting the projects ProjectController.